crates/re_viewport/src/viewport.rs

```diff
@@ -499,6 +507,36 @@ impl<'a, 'b> Viewport<'a, 'b> {
                     );
                     self.tree_edited = true;
                 }
+                TreeAction::MoveContentsToNewContainer {
+                    contents_to_move,
+                    new_container_kind,
+                    target_container,
+                    target_position_in_container,
+                } => {
+                    let new_container_tile_id = self
+                        .tree
+                        .tiles
+                        .insert_container(egui_tiles::Container::new(new_container_kind, vec![]));
+
+                    let target_container_tile_id = blueprint_id_to_tile_id(&target_container);
+                    self.tree.move_tile_to_container(
+                        new_container_tile_id,
+                        target_container_tile_id,
+                        target_position_in_container,
+                        true,
```

Comment:
an inline comment here as a form of poor-man's inlay hint would greatly improve the github experience :unamused: 
```suggestion
                        true, // what param is that